src/insights_client/__init__.py

```diff
@@ -398,13 +421,17 @@ class InsightsConstants(object):
 
         # handle client instantation here so that it isn't done multiple times in __init__
         # The config can be passed now by parameter
+        logger.debug("Initializing InsightsClient with the loaded config...")
         client = InsightsClient(config, False)  # read config, but dont setup logging
+        logger.debug("InsightsClient initialized. Egg version: %s" % (client.version(),))
 
         # we now have access to the clients logging mechanism instead of using print
+        logger.debug("Setting up regular logging.")
```

Comment:
This is not interesting enough to log.

Reply:
From a developer perspective, it may be. But it probably doesn’t deserve to be printed just for _verbose_.